balboa-client/src/main/scala/com/socrata/balboa/impl/HashMapBuffer.scala

```diff
@@ -8,8 +8,8 @@ import com.socrata.metrics.MetricQueue
 import com.socrata.metrics.components.{BufferComponent, BufferItem, MessageQueueComponent}
 import org.slf4j.LoggerFactory
 
+import collection.mutable
```

Comment:
Is there a reason to not include the `scala.` at the beginning of the package name here?